src/drivers/sdl/lv_sdl_mouse.c

```diff
@@ -35,6 +35,9 @@ typedef struct {
     int16_t last_x;
     int16_t last_y;
     bool left_button_down;
+#if LV_SDL_MOUSEWHEEL_MODE == 1
```

Comment:
I think using "constants" (in our case new `SD_MOUSEWHEEL_MODE_ENCODER` and `SDL_MOUSEWHELL_MODE_CROWN` macros) is better and more descriptive than repeating the values `0` and `1` in files.
Now we know what `0` and `1` mean, but later we will forget, and have to remember, or investigate what they mean.

Reply:
True, I'll fix these.